File: ANGULARPROJECTS/practica_angular01/src/app/Models/serie.ts
export default class Serie{
    public idSerie : number
    public nombre : string
    public imagen : string
    public puntuacion : number
    public anyo : number

    constructor(idSerie ?: number , nombre ?: string , imagen ?: string , puntuacion ?: number ,  anyo ?: number){
        this.idSerie = idSerie ?? NaN;
        this.nombre = nombre ?? '';
        this.imagen = imagen ?? '';
        this.puntuacion = puntuacion ?? NaN;
        this.anyo = anyo ?? NaN;
    }
}

File: .NET/AWS SQS/MvcCoreSeries/MvcCoreSeries/Models/Serie.cs
﻿using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace MvcCoreSeries.Models
{
    [Table("SERIES")]
    public class Serie
    {
        [Key]
        [Column("IDSERIE")]
        public int Id { get; set; }
        [Column("SERIE")]
        public string? Nombre { get; set; }
        [Column("IMAGEN")]
        public string? Imagen { get; set; }
        [Column("ANYO")]
        public int Anyo { get; set; }
    }
}
